renderdoc/os/win32/win32_shellext.cpp

```diff
@@ -510,6 +546,30 @@ struct RDCThumbnailProvider : public IThumbnailProvider, IInitializeWithStream
                       greenBlock[pixelInStride + (i * 4)];    // copy green channel
                 }
                 break;
+              case ResourceFormatType::BC6:
+                for(int pixelInStride = 0; pixelInStride < 4; pixelInStride++)
+                {
+                  // compute Pixel Index for the decompressedBC6 buffer. One block stide = 12
+                  // floats.
+                  int pixelIndex = pixelInStride * 3 + (i * 12);
+                  // decompressed BC6 block uses 16:16:16 color format. Convert to 8:8:8:8
+                  unsigned short r = decompressedBC6[pixelIndex];
+                  unsigned short g = decompressedBC6[pixelIndex + 1];
+                  unsigned short b = decompressedBC6[pixelIndex + 2];
```

Comment:
Same thing here please - prefer `uint16_t` sized format.

Reply:
Changed to `uint16_t`.